observability/micrometer/src/main/java/org/wildfly/extension/micrometer/MicrometerRegistryService.java

```diff
@@ -18,19 +18,22 @@
  */
 package org.wildfly.extension.micrometer;
 
+import static org.wildfly.extension.micrometer.MicrometerExtensionLogger.MICROMETER_LOGGER;
 import static org.wildfly.extension.micrometer.MicrometerSubsystemDefinition.MICROMETER_REGISTRY_RUNTIME_CAPABILITY;
 
-import java.io.IOException;
-import java.util.function.Consumer;
-import java.util.function.Supplier;
-
 import org.jboss.as.controller.CapabilityServiceBuilder;
 import org.jboss.as.controller.OperationContext;
 import org.jboss.msc.Service;
 import org.jboss.msc.service.StartContext;
 import org.jboss.msc.service.StopContext;
 import org.wildfly.extension.micrometer.jmx.JmxMicrometerCollector;
-import org.wildfly.extension.micrometer.metrics.WildFlyRegistry;
+import org.wildfly.extension.micrometer.registry.NoOpRegistry;
+import org.wildfly.extension.micrometer.registry.WildFlyOtlpRegistry;
+import org.wildfly.extension.micrometer.registry.WildFlyRegistry;
+
+import java.io.IOException;
+import java.util.function.Consumer;
+import java.util.function.Supplier;
```

Comment:
These were in the right spot before :)

Reply:
Grr. I had IDEA configured correctly at one point. :P